### ISSUE TITLE
Hourly join loop makes justlog crash

### ISSUE THREAD

**Issue by Fijxu** [gempir/justlog#183]
That it's
Any fixes for this?

```
Nov 26 14:04:20 localhost justlog[1887519]: time="2022-11-26T14:04:20-03:00" level=info msg="[bot] running hourly join loop"
Nov 26 14:04:33 localhost justlog[1887519]: runtime: goroutine stack exceeds 1000000000-byte limit
Nov 26 14:04:33 localhost justlog[1887519]: runtime: sp=0xc0207e0388 stack=[0xc0207e0000, 0xc0407e0000]
Nov 26 14:04:33 localhost justlog[1887519]: fatal error: stack overflow

... (Runtime stack that fills the logs)
```

**Comment by gempir** [comment]
No other useful log output? Haven’t heard of this issue before.

**Comment by Fijxu** [comment]
> No other useful log output? Haven’t heard of this issue before.

Runtime stack:
https://paste.zzls.xyz/ulesiviqir.yaml

**Comment by isabelcoolaf** [comment]
+1 same issue here on 2998fbd

**Comment by gempir** [comment]
Dumb mistake on my part, hopefully should be resolved with 8134c65b1304433bf731160f9311a866f05e905c

**Comment by Fijxu** [comment]
Fixed, justlog has been runnning for 2 days without any problems. Ty gempir :)
